docs/architecture/adr-043-interchain-identifiers.md

```diff
@@ -0,0 +1,109 @@
+# ADR 043: Interchain Identifiers (IID)
+
+## Context
+
+In the Internet of Blockchains we need a way to identify and locate assets within their origin chain namespace, using web-standard ([RFC39](https://datatracker.ietf.org/doc/html/rfc3986)) Universal Resource Indentifiers (URIs).
+
+When we refer to on-chain assets, this includes tokens, wallets, name records, or any other uniquely identifiable entity within a chain namespace.
+
+**Interchain Identifiers** (IIDs) are a standards-compliant method (based on [W3C DID-Core](https://w3c.github.io/did-core/)) for uniquely identifying, locating, referring to and verifying digital assets which are accessed within, or through, chain namespaces.
```

Comment:
let's make the explanation a bit easier, eg: _In other words, Cosmos IID is DID with an on chain record_.